core/src/test/java/com/google/googlejavaformat/java/MainTest.java

```diff
@@ -69,4 +74,28 @@ public void cantImportSortRangeYet() throws UsageException {
     assertThat(err.toString()).contains(
         "--sort-imports can currently only apply to the whole file");
   }
+
+  @Test
+  public void preserveOriginalFile() throws Exception {
+    Path from = Paths.get("src/test/resources/com/google/googlejavaformat/java/testdata/");
+    Path temp = testFolder.newFolder("preserve").toPath();
+    try (DirectoryStream<Path> stream = Files.newDirectoryStream(from, "*.output")) {
```

Comment:
But it's always good to know, that all generated/expected outputs are congruent with themselves, not? Is the duration too long? Or will it be some day, when there're 1000s of `*.output`files?